BoxContentSDK/BoxContentSDK/Cache/BOXRequestCache.m

```diff
@@ -0,0 +1,87 @@
+//
+//  BOXRequestCache.m
+//  BoxContentSDK
+//
+//  Created on 8/25/15.
+//  Copyright (c) 2015 Box. All rights reserved.
+//
+
+#import "BOXRequestCache.h"
+#import "BOXContentClient.h"
+#import "BOXUser.h"
+#import "BOXRequest_Private.h"
+#import "BOXFolderItemsRequest.h"
+
+#define CACHE_FOLDER @"boxRequestCache"
+
+@interface BOXRequestCache ()
+
+@property (nonatomic, readwrite, strong) NSString *cachePath;
```

Comment:
It's safer to use copy over strong for NSStrings: https://stackoverflow.com/questions/11249656/should-an-nsstring-property-under-arc-be-strong-or-copy